index.html

```diff
@@ -42,8 +42,168 @@ <h1>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>2015</span>
+        </div>
+        <p>
+          A daily selection of privately personalized reads;
+           no accounts or sign-ups required.
+        </p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <a class="see-project btn">See Project</a>
+      </div>
+
+      <div class="work-card card-2">
+        <div class="work-card-image-container">
+          <img src="./images/Snapshoot_2.jpg" alt="First work image">
+        </div>
+        <h2>Multi-Post Stories</h2>
+        <div class="work-card-properties-container">
+          <span>CANOPY</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>Back End Dev</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>2015</span>
+        </div>
+        <p>
+          A daily selection of privately personalized reads;
+           no accounts or sign-ups required.
+        </p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <a class="see-project btn">See Project</a>
+      </div>
+
+      <div class="work-card card-3">
+        <div class="work-card-image-container">
+          <img src="./images/Snapshoot_3.jpg" alt="First work image">
+        </div>
+        <h2>Tonic</h2>
+        <div class="work-card-properties-container">
+          <span>CANOPY</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>Back End Dev</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>2015</span>
+        </div>
+        <p>
+          A daily selection of privately personalized reads;
+           no accounts or sign-ups required.
+        </p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <a class="see-project btn">See Project</a>
+      </div>
+
+      <div class="work-card card-4">
+        <div class="work-card-image-container">
+          <img src="./images/Snapshoot_4.jpg" alt="First work image">
+        </div>
+        <h2>Multi-Post Stories</h2>
+        <div class="work-card-properties-container">
+          <span>CANOPY</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>Back End Dev</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>2015</span>
+        </div>
+        <p>
+          A daily selection of privately personalized reads;
+           no accounts or sign-ups required.
+        </p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <a class="see-project btn">See Project</a>
+      </div>
+
+    </section>
+
+    <!-- About Me Section -->
+    <section class="about">
+      <h1>About Myself</h1>
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make use of other `hx` tag apart from `h1` for semantic reasons stated by the previous reviewer 👍 👍